elastic/logs/tasks/many-shards-repo-setup.json

```diff
@@ -1,12 +0,0 @@
-{% set upd=p_snapshot_repo_settings.update({"base_path":"many-shards-"+((now|int)|string)}) %}
```

Comment:
Should you remove `    {% include "tasks/many-shards-setup.json" %},` in the many-shard-snapshots.json file?

Reply:
Hm not sure why? We need the tasks from there like https://github.com/elastic/rally-tracks/blob/7354a0307ae67463adabdde4fc1719f132f2ce80/elastic/logs/tasks/many-shards-setup.json#L2-L13

Reply:
Sorry I confused this file with *repo* setup!